README.md

```diff
@@ -19,10 +19,10 @@ the Linux Kernel Virtual Machine (KVM) to create and run microVMs. Firecracker
 has a minimalist design. It excludes unnecessary devices and guest-facing
 functionality to reduce the memory footprint and attack surface area of each
 microVM. This improves security, decreases the startup time, and increases
-hardware utilization. Firecracker currently supports Intel CPUs, with planned
-AMD and Arm support. Firecracker will also be integrated with popular container
-runtimes.
-
+hardware utilization. Firecracker currently supports Intel, AMD (beta) and Arm
```

Comment:
I propose using "public preview" or "preview" for the step between development and general availability. This is consistent with both our [roadmap](https://github.com/firecracker-microvm/firecracker/projects/13) and a more user-oriented view of things.